tests/infrastructure/node-labeller.go

```diff
@@ -276,7 +276,10 @@ var _ = DescribeInfra("Node-labeller", func() {
 		It("[test_id:6252] should remove all cpu model labels (all cpu model are in obsolete list)", func() {
 			node := nodesWithKVM[0]
 
-			obsoleteModels := nodelabellerutil.DefaultObsoleteCPUModels
+			obsoleteModels := map[string]bool{}
```

Comment:
nit: maybe a func returning the copy would make sense? And in the long run don't use the exported map, since it's mutable?

Reply:
A nice solution would be support of const but we don't have it. I think passing copies each time is also not elegant. I would leave it as is for lack of a better solution.